modules/integration/tests/src/test/java/org/wso2/appmanager/integration/test/cases/admin/dashboard/businessowner/UpdateBusinessOwnerTestCase.java

```diff
@@ -0,0 +1,86 @@
+/*
+*Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+package org.wso2.appmanager.integration.test.cases.admin.dashboard.businessowner;
+
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.appmanager.integration.utils.APPMAdminDashboardRestClient;
+import org.wso2.appmanager.integration.utils.AppmTestConstants;
+import org.wso2.appmanager.integration.utils.bean.BusinessOwner;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.automation.engine.context.beans.User;
+import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
+
+/**
+ * Test case which verifies the ability of admin user of updating a given business owner.
+ */
+public class UpdateBusinessOwnerTestCase {
+    private static final String TEST_DESCRIPTION = "Verify retrieving given business owner.";
+    private APPMAdminDashboardRestClient appmAdminDashboardRestClient;
+    private String businessOwnerName = "UpdateBusinessOwnerTestCase";
+    private static AutomationContext appMServerSuperTenant;
+
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/e263afed698e7dbd1253c0136563513fd4051eea